pages/application.tsx

```diff
@@ -83,185 +63,190 @@ const Application: NextPage = () => {
-						{Object.entries(ICEBREAKERS).map(([property, prompt]) => (
-							<TextInput required key={property} label={prompt} {...form.getInputProps(property)} withAsterisk />
-						))}
-
-						<Divider />
-
-						<MultiSelect
-							required
-							label="How did you hear about PickHacks?"
-							{...form.getInputProps('discoveryMethod')}
-							data={discoveryOptions}
-							searchable
-							creatable
-							getCreateLabel={(query) => query}
-							onCreate={(query) => {
-								const item = { value: query, label: query };
-								setDiscoveryOptions([...discoveryOptions, { value: query, label: query }]);
-								return item;
-							}}
-						/>
-						<Checkbox label="Please check if you will be attending our prehacks event." {...form.getInputProps('attendingPrehacks')} />
-						<Checkbox
-							required
-							label={
-								<>
-									I have read and agree to the{' '}
-									<a href="https://static.mlh.io/docs/mlh-code-of-conduct.pdf" target="_blank" rel="noopener noreferrer">
-										MLH Code of Conduct.
-									</a>
-								</>
-							}
-							{...form.getInputProps('codeOfConductAgreement')}
-						/>
-						<Checkbox
-							required
-							label={
-								<>
-									I authorize you to share my application/registration information for event administration, ranking, MLH administration,
-									pre- and post-event informational e-mails, and occasional messages about hackathons in-line with the MLH Privacy Policy. I
-									further I agree to the terms of both the
-									<a
-										href="https://github.com/MLH/mlh-policies/tree/master/prize-terms-and-conditions"
-										target="_blank"
-										rel="noopener noreferrer">
-										{' '}
-										MLH Contest Terms and Conditions
-									</a>{' '}
-									and the
-									<a href="https://mlh.io/privacy" target="_blank" rel="noopener noreferrer">
-										MLH Privacy Policy.
-									</a>
-								</>
+					<Stepper
+						active={step}
+						onStepClick={(step: number) => {
+							if (!form.validate().hasErrors) {
+								setStep(step);
 							}
-							{...form.getInputProps('dataAgreement')}
-						/>
-						<Checkbox
-							label="I authorize MLH to send me pre- and post-event informational emails, which contain free credit and opportunities from their partners."
-							{...form.getInputProps('mlhAgreement')}
-						/>
+						}}>
+						<Stepper.Step label="Personal Info">
+							<Box sx={{ maxWidth: 600 }} mx="auto">
+								<Stack>
+									<TextInput required label="First Name" {...form.getInputProps('firstName')} />
+									<TextInput required label="Last Name" {...form.getInputProps('lastName')} />
+									<TextInput required label="Email" placeholder="your@email.com" {...form.getInputProps('email')} />
+									<TextInput label="Phone Number" {...form.getInputProps('phoneNumber')} />
+									<NativeSelect required label="School" {...form.getInputProps('school')} data={['MST', 'Mizzou', 'Other School']} />
```

Comment:
We should consider just having this as a TextInput, Other School isn't very descriptive for our statistics. Alternatively, we could have what they had on the quill, which was an extremely large searchable Select with the options stored in a json file. Though, if we were to do that, it's probably large enough for its own branch and pull request.
TLDR: This field is better as an open TextInput than a very limited multiple choice.

Reply:
Yes I know I was the one who added it in the first place, I see the error of my ways

Reply:
i could make it a text input or i could alternatively add the ability to add schools to the dropdown, which do u think would be better?